src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ItemsRepeater.cs

```diff
@@ -845,12 +845,12 @@ void OnLayoutChanged(Layout oldValue, Layout newValue)
 			if (newValue != null)
 			{
 				newValue.InitializeForContext(GetLayoutContext());
-				newValue.MeasureInvalidated += InvalidateMeasureForLayout;
-				newValue.ArrangeInvalidated += InvalidateArrangeForLayout;
+				newValue.WeakMeasureInvalidated += InvalidateMeasureForLayout;
```

Comment:
Please add a comment `// Uno specific:` to make it clear that the code intentionally differs here

Reply:
Done.